test/unit/platforms/detect/os_windows_test.rb

```diff
@@ -104,4 +104,23 @@ def initialize
       detector.platform[:release].must_equal('4.10.1998')
     end
   end
+
+  describe 'windows 2019 Microsoft OpenSSH' do
+    let(:detector) {
+      detector = OsDetectWindowsTester.new
+      detector.backend.mock_command('cmd /c ver', "\'ver\"\' is not recognized as an internal or external command,\r\noperable program or batch file.", '', 1)
```

Comment:
Simulate the error with Microsoft's OpenSSH implementation. I haven't figured out why in that situation the command `ver"` is called, some quoting issue, but it can even be in their OpenSSH implementation.

In that case the code in lib/train/platforms/detect/helpers/os_windows.rb tries to run the command directly `ver`.